CMIOMinimalSample/Stream.mm

```diff
@@ -171,22 +171,16 @@ - (void)fillFrame {
 
     CVPixelBufferRef pixelBuffer = [self createPixelBufferWithTestAnimation];
 
-    // The timing here is quite important. For frames to be delivered correctly and successfully be recorded by apps
-    // like QuickTime Player, we need to be accurate in both our timestamps _and_ have a sensible scale. Using large
-    // timestamps and scales like mach_absolute_time() and NSEC_PER_SEC will work for display, but will error out
-    // when trying to record.
-    //
-    // Instead, we start our presentation times from zero (using the sequence number as a base), and use a scale that's
-    // a multiple of our framerate. This has been observed in parts of AVFoundation and lets us be frame-accurate even
-    // on non-round framerates (i.e., we can use a scale of 2997 for 29,97 fps content if we want to).
-    CMTimeScale scale = FPS * 100;
-    CMTime frameDuration = CMTimeMake(scale / FPS, scale);
-    CMTime pts = CMTimeMake(frameDuration.value * self.sequenceNumber, scale);
+    double hostTime = CACurrentMediaTime();
```

Comment:
What's the difference in `CACurrentMediaTime() * NSEC_PER_SEC` and `mach_absolute_time()`? Shouldn't they be the same?

Reply:
The accuracy is different.

Reply:
Why do you think that? From the Apple [docs](https://developer.apple.com/documentation/quartzcore/1395996-cacurrentmediatime?language=objc):

> A CFTimeInterval derived by calling mach_absolute_time() and converting the result to seconds.

Reply:
`#import <Foundation/Foundation.h>
#import <mach/mach_time.h>
#import <QuartzCore/CoreAnimation.h>

int main(int argc, char *argv[]) {
	@autoreleasepool {
		NSLog(@"%llu",(uint64_t)(CACurrentMediaTime()*NSEC_PER_SEC));
		NSLog(@"%llu",mach_absolute_time());
	}
}`

2020-05-09 05:56:12.165 Timer[11518:763703] 46073695407692
2020-05-09 05:56:12.166 Timer[11518:763703] 46073696399730

Reply:
It was my mistake.
The following code also worked correctly.
```
uint64_t hostTime = mach_absolute_time();
double timescale = 600.0;

CMTime eventTime = CMTimeMake((hostTime/(double)NSEC_PER_SEC)*timescale,timescale);
CMSampleTimingInfo timing;
timing.duration = CMTimeMake(timescale,FPS*timescale);
timing.presentationTimeStamp = eventTime;
timing.decodeTimeStamp = eventTime;
	   
OSStatus err = CMIOStreamClockPostTimingEvent(eventTime,hostTime,true,self.clock);
```
It was recorded correctly in QT Player.